backend/coins/btc/handlers/handlers.go

```diff
@@ -364,6 +364,52 @@ func (handlers *Handlers) getAccountBalance(*http.Request) (interface{}, error)
 	}, nil
 }
 
+type slip24Request struct {
+	RecipientName string `json:"recipientName"`
+	Nonce         string `json:"nonce"`
+	Memos         []struct {
+		Type         string `json:"type"`
+		Text         string `json:"text"`
+		Refund       string `json:"refund,omitempty"`
+		CoinPurchase struct {
+			CoinType int    `json:"coinType"`
+			Amount   string `json:"amount,omitempty"`
+			Address  string `json:"address,omitempty"`
+		} `json:"coinPurchase"`
+	} `json:"memos,omitempty"`
+	Outputs []struct {
+		Amount  float64 `json:"amount"`
+		Address string  `json:"address"`
+	} `json:"outputs,omitempty"`
+	Signature string `json:"signature,omitempty"`
+}
+
+func (slip24 slip24Request) toPaymentRequest() (*accounts.PaymentRequest, error) {
+	paymentRequest := &accounts.PaymentRequest{
+		RecipientName: slip24.RecipientName,
+		Nonce:         slip24.Nonce,
```

Comment:
I would put `Nonce: nil` here and return an error if slip24.Nonce is not empty, as we don't support it anyway.